src/components/CompareResults/CompareResultsView.tsx

```diff
@@ -11,10 +11,10 @@ import CompareResultsTable from './CompareResultsTable';
 function CompareResultsView(props: CompareResultsViewProps) {
   const { revisions, mode } = props;
   return (
-    <Container maxWidth="lg">
+    <Container maxWidth="xl">
```

Comment:
made the container wider so that the results table doesn't overflow and cause side-scrolling